lib/client/ui/plugins/camera.js

```diff
@@ -58,16 +58,14 @@ module.exports = {
             capture.setAttribute("controls",true);
             capture.style.display = "inline";
             capture.style.height = (screen.availHeight - 50) + "px";
-            capture.src = window.webkitURL.createObjectURL(upload.files[0]);
+            capture.src = (window.URL || window.webkitURL).createObjectURL(upload.files[0]);
```

Comment:
`window.webkitURL` is no longer needed in Chrome, it supports plain `window.URL`. tested on Chrome 42.